client/src/containers/App/App.js

```diff
@@ -34,21 +34,27 @@ class App extends Component {
           <Nav />
           <div id="page-wrapper">
             <Route exact path="/" component={Home} />
-            <Route path="/my-profile" component={MyProfile} />
-            <Route path="/help" component={Help} />
-            <Route path="/dashboard" component={Dashboard} />
-            <Route path="/courses" component={Courses} />
-            <Route path="/admin/organisations" component={Organisations} />
+            <Route exact path="/my-profile" component={MyProfile} />
```

Comment:
You don't need to add "exact" everywhere. It's useful just in the case a route prefix can also match other routes. So for example for admin/organisations (because it could also match partially admin/organisations/add).

Reply:
this is not my code

Reply:
if you dont want this I can delete just let me